frontend/src/pages/Search.js

```diff
@@ -124,24 +132,56 @@ function Search() {
   // way to display results
   const [isList, setView] = useState(isMobile ? false : true);
 
+  // sets search form state defaults in sessionStorage
+  setSSObject('select_sortby', sortbyOptions[0], true);
+  setSSObject(
+    'select_seasons',
+    [{ value: '202101', label: 'Spring 2021' }],
+    true
+  );
+  setSSObject('select_skillsareas', undefined, true);
+  setSSObject('select_credits', undefined, true);
+  setSSObject('select_schools', [], true);
+  setSSObject('select_subjects', [], true);
+  setSSObject('hideCancelled', true, true);
+  setSSObject('hideFirstYearSeminars', false, true);
+  setSSObject('ratingBounds', [1, 5], true);
+  setSSObject('workloadBounds', [1, 5], true);
+
   // react-select states for controlled forms
-  const [select_sortby, setSelectSortby] = useState(sortbyOptions[0]);
-  const [select_seasons, setSelectSeasons] = useState([
-    { value: '202101', label: 'Spring 2021' },
-  ]);
-  const [select_skillsareas, setSelectSkillsAreas] = useState();
-  const [select_credits, setSelectCredits] = useState();
-  const [select_schools, setSelectSchools] = useState([]);
-  const [select_subjects, setSelectSubjects] = useState([]);
+  const [select_sortby, setSelectSortby] = useState(
+    getSSObject('select_sortby')
+  );
```

Comment:
This entire thing feels really repetitive and that worries me

What if we added a `useSessionStorageState` React hook that takes a name and default value and manages all of this stuff internally
```JavaScript
function useSessionStorageState(name, default) {
  setSSObject(name, default, true);
  const [value, setValue] = useState(getSSObject(name));
  useEffect(() => {
    setSSObject(name)
  }, [value]);
  return [value, setValue];
}
```